app/code/Goeasyship/Shipping/Model/EasyshipCarrier.php

```diff
@@ -226,17 +226,16 @@ protected function _createEasyShipRequest(RateRequest $request)
     }
 
     /**
-     * Get weight
-     * @param $item
+     * Get website id
      * @return int
      */
-    protected function getStoreId()
+    protected function getWebsiteId()
     {
-        if (empty($this->storeId)) {
-            $this->storeId = $this->_storeManager->getStore()->getId();
+        if (empty($this->websiteId)) {
```

Comment:
will this break backward compatibility? Or this variable is also accessible on every versions~